src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextAuction.tsx

```diff
@@ -38,8 +40,8 @@ export const ArtworkContextAuctionQueryRenderer: React.SFC<
                 $excludeArtworkIDs: [String!]
                 $isClosed: Boolean!
               ) {
-                artwork(id: $artworkSlug) {
-                  ...ArtworkContextAuction_artwork
+                viewer {
```

Comment:
We needed to make this query spread on `Viewer`, and not `Artwork`, so we could also include the 'current sales' fragment.